### PR TITLE
Prefill more columns for Extraction and Transfer

### PR DESCRIPTION
**Pull request by nafiz1001**

### CODE REVIEW

File: backend/fms_core/templates.py [hunk 1]
```diff
@@ -489,7 +489,25 @@
       },
   ],
   "user prefill info": {
+      # "Extraction Type": ["DNA", "RNA"], # already prefilled by workflow
+      "Volume Used (uL)": "number",
+      # borrowed from extraction template
+      "Destination Container Kind": [
```
Comment (UlysseFG) [inline backend/fms_core/templates.py:495]: you should cast SAMPLE_CONTAINER_KINDS from containers.py into a list instead. The less places we have to update the same information the best.

File: backend/fms_core/templates.py [hunk 2]
```diff
@@ -515,7 +533,23 @@
       },
   ],
   "user prefill info": {
+      # borrowed from transfer template
+      "Destination Container Kind": [
+        "Tube",
```
Comment (UlysseFG) [inline backend/fms_core/templates.py:538]: you should cast SAMPLE_CONTAINER_KINDS from containers.py into a list instead. The less places we have to update the same information the best.

File: backend/fms_core/templates.py (hunk 1 above)
Comment (nafiz1001) [reply backend/fms_core/templates.py:495]: i'm not quite sure how it applies in here. i picked only the containers that are relevant to sample extraction. is this just for improving existing codes?

Comment (UlysseFG) [reply backend/fms_core/templates.py:495]: It applies that w can typically extract in any container that can receive samples. These are listed in the above mentioned constant. This is meant to make the code require less maintenance. This list will be automatically updated with new values as new containers are created while your hard coded list will need to be updated specifically each time.

File: backend/fms_core/templates.py [hunk 3]
```diff
@@ -489,7 +490,14 @@
       },
   ],
   "user prefill info": {
+      # "Extraction Type": ["DNA", "RNA"], # already prefilled by workflow
+      "Volume Used (uL)": "number",
+      # borrowed from extraction template
+      "Destination Container Kind": list(SAMPLE_NON_RUN_CONTAINER_KINDS),
+      "Volume (uL)": "number",
+      "Source Depleted": ["YES", "NO"],
```
Comment (UlysseFG) [inline backend/fms_core/templates.py:498]: Remove the "NO" from the list there is really just an option here...

File: backend/fms_core/templates.py [hunk 4]
```diff
@@ -515,7 +523,12 @@
       },
   ],
   "user prefill info": {
+      # borrowed from transfer template
+      "Destination Container Kind": list(SAMPLE_NON_RUN_CONTAINER_KINDS),
+      "Source Depleted": ["YES", "NO"],
```
Comment (UlysseFG) [inline backend/fms_core/templates.py:528]: Remove the "NO" from the list there is really just an option here...

**Review by UlysseFG — APPROVED** [review]

**Pull request merged**